ink-wrapper/src/extensions.rs

```diff
@@ -59,15 +59,15 @@ impl TypeExtensions for Type<PortableForm> {
         !self.is_primitive() && !self.is_ink() && !self.is_builtin()
     }
 
-    fn qualified_name(&self) -> String {
+    fn qualified_name(&self) -> proc_macro2::TokenStream {
         if self.is_lang_error() {
-            "ink_wrapper_types::InkLangError".to_string()
+            quote::quote! { ink_wrapper_types::InkLangError }
         } else if self.is_ink_types() {
-            ["ink_primitives", self.path.segments.last().unwrap()].join("::")
-        } else if self.is_ink() {
```

Comment:
Removed this branch since `is_ink_types()` comes first and it already catches what `is_ink()` would.